src/current/cockroachcloud/regions.md

```diff
@@ -77,10 +77,6 @@ Western Europe  | `eu-central-1`   | Frankfurt
 
 ## Azure regions
 
-{{site.data.alerts.callout_info}}
-{% include feature-phases/azure-limited-access.md %}
```

Comment:
Can [`azure-limited-access.md`](https://github.com/cockroachdb/docs/blob/main/src/current/_includes/feature-phases/azure-limited-access.md) be removed entirely, now?